utils/sql/src/main/java/io/apicurio/registry/storage/impl/sql/SqlGlobalIdGenerator.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.apicurio.registry.storage.impl.sql;
+
+/**
+ * @author Fabian Martinez
+ */
+public class SqlGlobalIdGenerator implements GlobalIdGenerator {
```

Comment:
To avoid a SQL injection vector, this could instead actually make a call to the DB to get the next value and return it.